docs/book/src/cronjob-tutorial/testdata/project/.dockerignore

```diff
@@ -1,11 +1,3 @@
 # More info: https://docs.docker.com/engine/reference/builder/#dockerignore-file
-# Ignore everything by default and re-include only needed files
-**
-
-# Re-include Go source files (but not *_test.go)
-!**/*.go
-**/*_test.go
-
-# Re-include Go module files
-!go.mod
-!go.sum
+# Ignore build and test binaries.
+bin/
```

Comment:
This file should not be changed
Can you please revert and ensure that the PR has only the required changes?